### PR TITLE
TICK_SIZE - bitrue

### PR DESCRIPTION
**Pull request by ttodua**

### CODE REVIEW

File: js/bitrue.js [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 const Exchange = require ('./base/Exchange');
 const { ExchangeError, ArgumentsRequired, ExchangeNotAvailable, InsufficientFunds, OrderNotFound, InvalidOrder, DDoSProtection, InvalidNonce, AuthenticationError, RateLimitExceeded, PermissionDenied, BadRequest, BadSymbol, AccountSuspended, OrderImmediatelyFillable, OnMaintenance } = require ('./base/errors');
-const { TRUNCATE } = require ('./base/functions/number');
+const { TRUNCATE, DECIMAL_PLACES } = require ('./base/functions/number');
```
Comment (kroitor) [inline js/bitrue.js:7]: Why do we use DECIMAL_PLACES instead of TICK_SIZE here?

Comment (ttodua) [reply js/bitrue.js]: I am surprised too  :D 
fixed.

File: js/bitrue.js [hunk 2]
```diff
@@ -619,10 +624,12 @@ module.exports = class bitrue extends Exchange {
...
             const filtersByType = this.indexBy (filters, 'filterType');
             const status = this.safeString (market, 'status');
-            const priceDefault = this.safeInteger (market, 'pricePrecision');
-            const amountDefault = this.safeInteger (market, 'quantityPrecision');
             const priceFilter = this.safeValue (filtersByType, 'PRICE_FILTER', {});
             const amountFilter = this.safeValue (filtersByType, 'LOT_SIZE', {});
+            const priceDefault = this.safeString (market, 'pricePrecision');
+            const amountDefault = this.safeString (market, 'quantityPrecision');
+            const amountPrecisionDigits = this.safeString (amountFilter, 'volumeScale', amountDefault);
+            const pricePrecisionDigits = this.safeString (amountFilter, 'priceScale', priceDefault);
```
Comment (kroitor) [inline js/bitrue.js:632]: Shouldn't we use the priceFilter here?

Comment (ttodua) [reply js/bitrue.js]: Good catch! now ready.

**Pull request merged**